src/sql/parts/notebook/notebook.component.html

```diff
@@ -7,7 +7,7 @@
 <div style="overflow: hidden; width: 100%; height: 100%; display: flex; flex-flow: column">
 	<div #toolbar class="editor-toolbar actionbar-container" style="flex: 0 0 auto; display: flex; flex-flow: row; width: 100%; align-items: center; height: 36px">
 	</div>
-	<div class="scrollable" style="flex: 1 1 auto; position: relative">
+	<div class="scrollable" style="flex: 1 1 auto; position: relative" (click)="unselectActiveCell()">
 		<loading-spinner [loading]="isLoading"></loading-spinner>
 		<div class="notebook-cell" *ngFor="let cell of cells" (click)="selectCell(cell)" [class.active]="cell.active" (keydown)="onKeyDown($event)">
```

Comment:
Quick question - does the `selectCell` click handler stop the click event bubbling up to the parent? I'm not super familiar with click hander behavior but I know some events keep propagating unless we stop them. Otherwise I'd expect things to go "click -> cell selected -> unselectActiveCell".

Reply:
This is answered below. Now using event.stopPropagation() as you suggested, which seems to work well for this case.